resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/submit/LoggingPodStatusWatcher.scala

```diff
@@ -28,8 +29,10 @@ import org.apache.spark.SparkException
 import org.apache.spark.internal.Logging
 import org.apache.spark.util.ThreadUtils
 
+
 private[k8s] trait LoggingPodStatusWatcher extends Watcher[Pod] {
-  def awaitCompletion(): Unit
+  def reset(): Unit
+  def watchOrStop(sId: String): Boolean
```

Comment:
Could you preserve the order from the [original patches](https://github.com/apache/spark/pull/28423/files#diff-ff85a8d70a121a39181ed5ddb1005f87bf677acbe0849defc9fc843bcef62df2R31) (master/branch-3.0)?
```scala
def watchOrStop(submissionId: String): Boolean
def reset(): Unit
```

The function declaration order is different. The parameter name is not the same here (sId != submissionId)

Reply:
Done.